Dockerfile.goreleaser

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.8
+
```

Comment:
Adding,

```Dockerfile
RUN apk add --no-cache curl
```

here fixes the issue with missing certs, from my testing at least.

Reply:
really? That's great!